src/test/java/org/gridsuite/ds/server/controller/DynamicSimulationControllerTest.java

```diff
@@ -564,12 +571,12 @@ public void testStopLately() throws Exception {
                 .containsEntry(HEADER_RESULT_UUID, runUuid.toString());
 
         // Must have a cancel message in the stop queue
-        message = output.receive(1000, dsStoppedDestination);
+        message = output.receive(1000, dsCancelFailedDestination);
         assertThat(message.getHeaders())
                 .containsEntry(HEADER_RESULT_UUID, runUuid.toString())
-                .containsEntry(HEADER_MESSAGE, getCancelMessage(COMPUTATION_TYPE));
+                .containsEntry(HEADER_MESSAGE, getCancelFailedMessage(COMPUTATION_TYPE));
         // result has been deleted by cancel so not found
-        assertNotFoundResult(runUuid);
+        assertResultStatus(runUuid, status().isOk());
```

Comment:
to discuss later, normally when user do stop, the result should be cleaned

Reply:
indeed to discuss later, we decided to change it so now if the cancel failed we don't clear the results because we already did all the computation so we can at least let the user decide if he wants to check them (he can still remove the result be redoing it)

Reply:
OK